docs/book/src/providers.md

```diff
@@ -37,3 +37,12 @@ To implement a secrets-store-csi-driver provider, you can develop a new provider
 - Provider mounts `<kubelet root dir>/pods` (default: [`/var/lib/kubelet/pods`](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/v0.0.14/deploy/secrets-store-csi-driver.yaml#L86-L87)) with [`HostToContainer` mount propagation](https://kubernetes-csi.github.io/docs/deploying.html#driver-volume-mounts) to be able to write the external secrets store content to the volume target path
 
 See [design doc](https://docs.google.com/document/d/10-RHUJGM0oMN88AZNxjOmGz0NsWAvOYrWUEV-FbLWyw/edit?usp=sharing) for more details.
+
+## Features supported by current providers
+
+| Features \ Providers               | Azure | GCP   | AWS   | Vault |
+| ---------------------------------- | ----- | ----- | ----- | ----- |
+| Sync as Kubernetes secret          | Yes   | Yes   | Yes   | Yes   |
+| Rotation                           | Yes   | No    | Yes   | No    |
```

Comment:
GCP should support rotation, is there some criteria for this that's missing?

Reply:
I populated this initial matrix by looking at e2e tests. I'll update this for GCP. Thanks for confirming.

Reply:
Ah thanks, yes and I've gone ahead and added an integration test for rotation to our provider-repo integration tests. I do not want to add one to this repo as that would require different IAM permissions